kroxylicious/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -250,6 +251,11 @@ public void initiateConnect(HostPort remote, List<FilterAndInvoker> filters) {
         Channel outboundChannel = connectFuture.channel();
         ChannelPipeline pipeline = outboundChannel.pipeline();
 
+        // Note: Because we are acting as a client of the target cluster and are thus writing Request data to an outbound channel, the Request flows from the
+        // last outbound handler in the pipeline to the first (i.e. In the reverse order of the configuration). When Responses are read from the cluster
```

Comment:
"In the reverse order of the configuration", I think you are talking about Kroxylicious filter order here.  I found it confusing as initially thought you were referring to the whole pipeline.

Reply:
I've move these statements about configuration out into a separate sentence to make it clearer, so later on it says `This is also the opposite of the order that Filters are declared in the Kroxylicious configuration`.